blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/astyanax/AstyanaxShardStateIO.java

```diff
@@ -0,0 +1,82 @@
+package com.rackspacecloud.blueflood.io.astyanax;
+
+import com.codahale.metrics.Timer;
+import com.netflix.astyanax.ColumnListMutation;
+import com.netflix.astyanax.MutationBatch;
+import com.netflix.astyanax.connectionpool.exceptions.ConnectionException;
+import com.netflix.astyanax.model.Column;
+import com.netflix.astyanax.model.ColumnList;
+import com.rackspacecloud.blueflood.io.*;
+import com.rackspacecloud.blueflood.rollup.Granularity;
+import com.rackspacecloud.blueflood.service.SlotState;
+import com.rackspacecloud.blueflood.service.UpdateStamp;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.Collection;
+import java.util.LinkedList;
+import java.util.Map;
+
+public class AstyanaxShardStateIO implements ShardStateIO {
```

Comment:
I don't see a big reason to make it a singleton. It is slightly harder to mock a singleton class. But I know how to do it now (I did it for the other singleton classes like AstyanaxIO, etc). You think it's better as singleton?